.gitignore

```diff
@@ -8,7 +8,6 @@ __pycache__
 *.egg
 *.egg-info
 dist
-build
```

Comment:
Please don't remove the line, just use `/build/` as in the example [here](https://github.com/overhangio/tutor-notes/blob/master/.gitignore#L6)